### DISCUSSION TITLE
slide show: adjusting time display

### DISCUSSION THREAD

**Discussion by jalbrechtDE** [LycheeOrg/Lychee#2476] [Q&A] [answered]
Hi folks,

my first-time-installation was easy and went fine, my first album is up and running ...

https://billerbecker-bilderbogen.medienflotte.de/frame/n2ojIBuYZ3Ma9Um-DAfhTppg

Now I am looking for a possibility to adjust **the time** the pictures are displayed during the slide show ... it seems quite long, almost 10 seconds ...

Thanks for a hint _(something I must have overlooked)_

joachim

**Comment by ildyria** [comment, marked as answer]
Go to your advanced settings (`Settings` => bottom => `More`).
Then search for `mod_frame_refresh`.
Change the value and click on the `Save blablabla` button at the bottom of the page (or just press `Enter`).

**Reply by jalbrechtDE** [reply]
I appreciate your the quick and helpful reply ... 

Have a nice weekend ... joachim
